<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HDiff_MainWindow</class>
 <widget class="QMainWindow" name="HDiff_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HDiff_MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="hViewer" native="true">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>0</y>
      <width>661</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>741</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile_F">
    <property name="title">
     <string>File(F)</string>
    </property>
    <addaction name="openAction"/>
    <addaction name="saveAction"/>
    <addaction name="saveAsAction"/>
    <addaction name="separator"/>
    <addaction name="printAction"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menuProcessing_P">
    <property name="title">
     <string>Processing(P)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTool_T">
    <property name="title">
     <string>Tool(T)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView_V">
    <property name="title">
     <string>View(V)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp_H">
    <property name="title">
     <string>Help(H)</string>
    </property>
   </widget>
   <addaction name="menuFile_F"/>
   <addaction name="menuProcessing_P"/>
   <addaction name="menuTool_T"/>
   <addaction name="menuView_V"/>
   <addaction name="menuHelp_H"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <action name="openAction">
   <property name="icon">
    <iconset resource="imgrc.qrc">
     <normaloff>:/viewer/img/open.png</normaloff>:/viewer/img/open.png</iconset>
   </property>
   <property name="text">
    <string>Load(L)</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="icon">
    <iconset resource="imgrc.qrc">
     <normaloff>:/viewer/img/save.png</normaloff>:/viewer/img/save.png</iconset>
   </property>
   <property name="text">
    <string>Save(S)</string>
   </property>
  </action>
  <action name="saveAsAction">
   <property name="text">
    <string>Save As(A)</string>
   </property>
  </action>
  <action name="printAction">
   <property name="text">
    <string>Print(P)</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>Exit(E)</string>
   </property>
  </action>
  <action name="mriAction">
   <property name="text">
    <string>MRI Processing</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="imgrc.qrc"/>
 </resources>
 <connections/>
</ui>
